calc.c

```diff
@@ -1,54 +0,0 @@
-#include<stdio.h>
-int main (){
-    float key;
-    printf("CALCULATOR BY SAKSHAM  \nUSER MANUAL: PRESS KEYS ACCORDINGLY \nTO ADD PRESS 1\nTO SUBTRACT PRESS 2\nTO MULTIPLY PRESS 3\nTO DIVIDE PRESS 4\nTO LEAVE PRESS 5 \nPRESS KEY (1-5) = ");
```

Comment:
learn about writting tests or unit test cases it will help overall to build a continuous code pipeline